<launch>

    <node name="pozyx_node" pkg="pozyx_drivers" type="pozyx_driver_ros.py" output="screen">
        <!-- POZYX_POS_ALG_UWB_ONLY = 0   POZYX_POS_ALG_TRACKING = 1   POZYX_POS_ALG_LS = 2 -->
    	<param name="algorithm" value="2" />
        <!-- POZYX_3D = 3    POZYX_2D = 2    POZYX_2_5D = 1 -->
    	<param name="dimension" value="3" />
        <!-- Height of the tag, required in 2.5D positioning -->
    	<param name="height" value="1000" />
    	<param name="frequency" value="10" />

		<param name="anchor0_id" value="0x6828" />
		<param name="anchor1_id" value="0x683D" />
		<param name="anchor2_id" value="0x6842" />
		<param name="anchor3_id" value="0x685C" />

    	<param name="anchor0_coordinates" value="0, 0, 2890" />
    	<param name="anchor1_coordinates" value="5295, 0, 620" />  <!--1577-->
    	<param name="anchor2_coordinates" value="0, 9630, 2116" />
    	<param name="anchor3_coordinates" value="5295, 9377, 3191" />
    </node>

</launch>
